summon/custom-panel-sections/ccac-archives.html

```diff
@@ -0,0 +1,23 @@
+<h3 ng-bind="::section.title" class="ng-binding">CCA/C Archives</h3>
+
+<section>
+    <img src="https://vault.cca.edu/file/bbede28d-4484-416e-a584-e82b9fff5c46/1/161014003.jpg" alt="Architects' preliminary model of the campus from CCAC Bulletin, 1965" style="float: left; padding: 0em 1em 0em 0em; width: 220;" class="randomImage">
```

Comment:
try to take some of the CSS in these `style` attributes and put it into the `randomImage` class you added, it's better to manage repeated styles (e.g. `float: left` is in every one) that way and only use the style attribute as a last resort for specific tweaks